### PR TITLE
Negativity Fly and Speed

### PR DESCRIPTION
**Pull request by DieSturmkatze**

### CODE REVIEW

File: runclient.bat
Comment (be4dev) [inline runclient.bat:1]: remove such files from pr

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/fly/modes/negativity/FlyNegativity.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ *
+ *
```
Comment (be4dev) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/fly/modes/negativity/FlyNegativity.kt:19]: remove those empty lines to make it standardized

File: .gitignore
Comment (Snowiiii) [inline .gitignore:1]: Please remove this nonsense

File: cmd.bat
Comment (Snowiiii) [inline cmd.bat:1]: Please remove this nonsense

File: runclient.bat
Comment (Snowiiii) [inline runclient.bat:1]: Please remove this nonsense

**Review by Snowiiii — COMMENTED** [review]
1: This is definitely the wrong PR for this
2. Its just nosense

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/fly/modes/negativity/FlyNegativity.kt (hunk 1 above)
Comment (1zun4) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/fly/modes/negativity/FlyNegativity.kt:19]: My IntellIJ does this as well, since it's caused by the copyright header feature.

**Comment by EclipsesDev** [comment]
code looks good, except those unnecessary stuff.. 🤔

**Comment by 1zun4** [comment]
Please undraft your Pull Request if you want it to be merged.

File: cmd.bat [hunk 2]
```diff
@@ -0,0 +1 @@
+cmd.exe
```
Comment (EclipsesDev) [inline cmd.bat:1]: Why is this necessary?

**Review by EclipsesDev — COMMENTED** [review]
umm

File: runclient.bat [hunk 3]
```diff
@@ -0,0 +1,2 @@
+gradlew :runClient --info &pause
```
Comment (EclipsesDev) [inline runclient.bat:1]: this too..

**Comment by EclipsesDev** [comment]
hmm @1zun4, i think u might don't want to add this, its useless anyway, since literally almost all fly/speed (even vanilla) will bypass negativity, tho if u want to add it, thats up to u

**Comment by Snowiiii** [comment]
I suggest to close this weird PR

**Comment by EclipsesDev** [comment]
> I suggest to close this weird PR

I guess so

**Comment by EclipsesDev** [comment]
feel free to reopen, if there's update about this pr
